### PR TITLE
DAC6-2253 : Added details to readme

### PR DESCRIPTION
**Pull request by AashuChauhanHmrc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
\ No newline at end of file
+## Info
+
+This service allows a user(agent, client or organisation) to register to CBCR services.
+
+This service has a corresponding back-end service, namely register-country-by-country-reporting can be viewed [here](https://github.com/hmrc/register-country-by-country-reporting) which integrates with HOD, i.e. DES/ETMP.
+
+### Dependencies
+
+| Service          | Link                                                                   |
```
Comment (matthewharris1) [inline README.md:11]: Please add other cbc services (cbc file upload frontend,backend and stubs AND cbc register backend and stubs)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,48 @@
...
+### Origin
+
+https://github.com/hmrc/register-country-by-country-reporting-frontend 
+
+### Endpoints used
+
+| Service              | HTTP Method | Route                                                                         | Purpose                                                          |
+|----------------------|-------------|-------------------------------------------------------------------------------|------------------------------------------------------------------|
+| Tax Enrolments       | POST        | /tax-enrolments/service/:serviceName/enrolment                                | Enrols a user synchronously for a given service name             
+| Registration         | POST        | /register-country-by-country-reporting/registration/noId                      | Enables user to register witout id
```
Comment (matthewharris1) [inline README.md:27]: The service being used is Registration backend not registration "register-country-by-country-reporting"

File: README.md [hunk 3]
```diff
@@ -1,3 +1,48 @@
...
+| Registration         | POST        | /register-country-by-country-reporting/registration/noId                      | Enables user to register witout id                                                                        
+| Registration         | POST        | /register-country-by-country-reporting/registration/utr                       | Enables user to register                                                                        
+| Subscription`        | POST        | /register-country-by-country-reporting/subscription/create-subscription       | Enables user to create subscription                                                                     
+| Subscription         | POST        | /register-country-by-country-reporting/subscription/read-subscription/:safeId | Enables user to read subscription details                                                                        
+| Email                | POST        | /hmrc/email                                                                   | Sends an email to an email address
```
Comment (matthewharris1) [inline README.md:31]: You've missed the enrolmentstoreproxy connector

**Review by matthewharris1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AashuChauhanHmrc) [reply README.md]: added mentioned services.

File: README.md (hunk 2 above)
Comment (AashuChauhanHmrc) [reply README.md]: Mentioned service as backend service , little unsure if this is what you were asking.

File: README.md (hunk 3 above)
Comment (AashuChauhanHmrc) [reply README.md]: Added proxy

File: README.md [hunk 4]
```diff
@@ -1,3 +1,54 @@
...
+|------------------------------------------------------|-------------------------------------------------------------------------| 
+| Address Lookup                                       | https://github.com/hmrc/address-lookup                                  |
+| Email                                                | https://github.com/hmrc/email                                           |
+| Auth                                                 | https://github.com/hmrc/auth                                            |
+| Tax Enrolments                                       | https://github.com/hmrc/tax-enrolments                                  |
+| Country by country reporting frontend                | https://github.com/hmrc/country-by-country-reporting-frontend           |
+| Country by country reporting                         | https://github.com/hmrc/country-by-country-reporting-frontend           |
+| Country by country reporting stubs                   | https://github.com/hmrc/country-by-country-reporting-stubs              |
+| Register country by country reporting frontend stubs | https://github.com/hmrc/register-country-by-country-reporting-stubs     |
```
Comment (matthewharris1) [inline README.md:20]: Register country-by-country-frontend and register-country-by-country-stubs are 2 different services. You only need to call stubs here.

File: README.md (hunk 2 above)
Comment (matthewharris1) [reply README.md]: image
All of these should have "Register-country-by-country-reporting" under the service section of the table

**Review by matthewharris1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (AashuChauhanHmrc) [reply README.md]: Sorry got it now ,  fixed it.

File: README.md (hunk 2 above)
Comment (AashuChauhanHmrc) [reply README.md]: Fixed.

**Review by matthewharris1 — APPROVED** [review]

**Pull request merged**